### ISSUE TITLE
Defender 2.2 Policy should be set to "block everyone"

### ISSUE THREAD

**Issue by adhilto** [cisagov/ScubaGear#123]
# 🐛 Summary #

Per the baseline, "The action for the DLP policy SHOULD be set to block sharing sensitive information with everyone when DLP conditions are met."

That means, the action for a given DLP rule should look like this:
![image](https://user-images.githubusercontent.com/106177711/213585122-7a392c69-ca9d-4a7e-bb82-1c2ffe9b41f0.png)

However, the code currently also accepts "Block only people outside your organization."

## To reproduce ##

Follow the instructions [here](https://github.com/cisagov/ScubaGear/blob/main/baselines/defender.md#224-implementation), but select "Block only people outside your organization", then run ScubaGear. Note that the test still passes.

## Expected behavior ##
The relevant variables the Rego should check are `BlockAccess` and `BlockAccessScope`. The code already asserts that `BlockAccess` is set to true, but `BlockAccessScope` needs to be checked as well. It will be set to `All` if "Block everyone" was selected and "PerUser" if "Block only people outside your organization" was selected.

**Comment by tkol2022** [comment]
Does this bug produce any known false positives or negatives at this time?

**Comment by adhilto** [comment]
Yes. The code will currently mark the test as passed even if "Block only people outside your organization" instead of "Block everyone" is selected.
